node/src/p2p.rs

```diff
@@ -146,21 +145,18 @@ impl From<oneshot::error::RecvError> for P2pError {
 
 /// Component responsible for the peer to peer networking handling.
 #[derive(Debug)]
-pub struct P2p<S>
-where
-    S: Store + 'static,
-{
+pub struct P2p {
```

Comment:
In the end, I'm going to need `Store` when I remove `async` from `Multihasher`, so I think we should keep it.

Reply:
won't it be needed only in `Worker`?